discovery/README.md

```diff
@@ -22,30 +31,96 @@ cd <some_path>/ansible_collections/confluent/platform
 PYTHONPATH=. python discovery/main.py --input discovery/hosts.yml [optional arguments] 
 ```
 #### Sample input file (hosts.yml)
+A Cluster with un-known host group mappings. In this case, the script will try to map the service and corresponding hosts.
 ```yaml
-all:
-  vars:
-    ansible_connection: ssh
-    ansible_become: true
-    ansible_python_interpreter: auto
-    ansible_user: centos
-    ansible_become_user: root
-    ansible_become_method: sudo
-    ansible_ssh_extra_args: -o StrictHostKeyChecking=no
-    ansible_ssh_private_key_file: <path_to_private_key_to_login_to_vms>
-  hosts:
-    - ec2-35-164-166-99.us-west-2.compute.amazonaws.com
-    - ec2-35-164-166-99.us-west-2.compute.amazonaws.com
-    - ec2-35-86-106-150.us-west-2.compute.amazonaws.com
-    - ec2-54-191-208-245.us-west-2.compute.amazonaws.com
-    - ec2-35-164-166-99.us-west-2.compute.amazonaws.com
+vars:
+  ansible_connection: ssh
+  ansible_user: centos
+  ansible_become: true
+  ansible_ssh_extra_args: -o StrictHostKeyChecking=no
+  ansible_ssh_private_key_file: ~/Work/keys/muckrake.pem
+
+hosts:
+  all:
+    - ec2-35-85-206-158.us-west-2.compute.amazonaws.com
```

Comment:
Unrelated note:
@nsharma-git 
Should we use some dummy name and not actual keyname and ec2 addresses? 
Atleast, for cp-ansible docs if not for the readme.